nexus-sdk-js/src/Project/index.ts

```diff
@@ -128,7 +127,8 @@ export default class Project {
     }
   }
 
-  async listSparqlViews(): Promise<SparqlView[]> {
+  // TODO: refactor once we can fetch views per IDs (broken just now)
+  async getSparqlView(): Promise<SparqlView> {
```

Comment:
Since the SPARQL view will always have a view URL that is /views/:org/:project/graph/sparql, can we use that to save an HTTP query?

Reply:
I'd rather call the api so it returns the view and its metadata for now. Maybe we can improve it in the future if it turns out to always return the same metadata to all users, no matter the permissions, etc...

Reply:
OK